packages/lib-utils/package.json

```diff
@@ -24,12 +24,13 @@
     "test": "yarn run -T test $INIT_CWD"
   },
   "peerDependencies": {
-    "@openshift/dynamic-plugin-sdk": "^1.0.0",
+    "@openshift/dynamic-plugin-sdk": ">=1.0.0-alpha",
```

Comment:
Does this help us get around the entire alpha versioning issue (where carat does not correctly pick up `alpha1` vs `alpha2` automatically)? If so, we should probably update hac-core so we don't need to change this manually each time we merge/publish the SDK.

Reply:
Yes, I think this should help us with the issue where pre-release versions are not picked up by regular caret `^` or tilde `~` ranges 😃 

As per SemVer rules, version `1.0.0-alpha1` comes "before" version `1.0.0` because it's a pre-release version. Therefore, version `1.0.0-alpha1` may not match the constraint `^1.0.0`.

In version range `>=1.0.0-alpha`, the explicit `-alpha` should cause it to pick up alpha pre-releases only. According to SemVer rules, pre-release identifiers are compared alpha numerically.

Reply:
I think ideally I'd like to remove the `alpha` postfix all-together. We can do that as a follow up.